### PR TITLE
adds table variants

### PR DESCRIPTION
**Pull request by jolamar**

### CODE REVIEW

File: src/components/Table/Table.tsx [hunk 1]
```diff
@@ -3,17 +3,19 @@ import * as React from 'react';
 import * as Rivet from '../util/Rivet';
 
 interface TableProps {
-    /** 
+    /**
      * Optional Rivet style which changes the look and feel of the rendered table.
      */
-    variant?: 'stripes' | 'plain'
+    variant?: 'stripes' | 'plain' | 'compact' | 'cells'
```
Comment (burnumd) [inline src/components/Table/Table.tsx]: Are these variants actually mutually exclusive? From the documentation, it sounds like either could be applied to a striped or plain table (or alongside each other).

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -3,17 +3,19 @@ import * as React from 'react';
...
 }
 
-const Table : React.SFC<TableProps & React.HTMLAttributes<HTMLTableElement>> = 
+const Table : React.SFC<TableProps & React.HTMLAttributes<HTMLTableElement>> =
 ({ children, className, variant, ...attrs }) => {
     const classes = classNames({
         ['rvt-table-plain']: variant === 'plain',
         ['rvt-table-stripes']: variant === 'stripes',
+        ['rvt-table-compact']: variant === 'compact',
```
Comment (jwbennet) [inline src/components/Table/Table.tsx]: You might change classes to the following:
```
const classes = classNames({
  [`rvt-table-${variant}`]: variant
});
```

I think having plain and stripes separate the way they are is a result of me copying a component where those were both boolean properties and I never merged them together correctly.

File: src/components/Table/Table.tsx (hunk 1 above)
Comment (levimcg) [reply src/components/Table/Table.tsx]: @burnumd That's a good call. Yes, they design so that they can be applied alongside each other if needed.

**Comment by jwbennet** [comment]
As a general comment, you may also modify the title of this pull request since it appears to be a catch-all for Rivet 1.1 updates.

File: src/components/Badge/Badge.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ interface BadgeProps {
     /**
      * Optional Rivet style: an action/error/success/warning badge.
      */
-    variant?: 'action' | 'error' | 'success' | 'warning';
+    variant?: 'action' | 'error' | 'danger' | 'success' | 'warning';
```
Comment (jwbennet) [inline src/components/Badge/Badge.tsx]: I think we should go ahead and remove the `error` variant as long as we mention it in the commit message for the release notes.

File: src/components/Alert/common.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ export interface StatefulAlertProps {
      * Rivet alert styling. 
      * @see https://rivet.uits.iu.edu/components/overlays/alerts
      */ 
-    variant: "error" | "info" | "message" | "success",
+    variant: "error" | "danger" | "info" | "message" | "success",
```
Comment (jwbennet) [inline src/components/Alert/common.ts]: I think we should go ahead and remove the `error` variant as long as we mention it in the commit message for the release notes.

File: src/components/Alert/common.ts [hunk 5]
```diff
@@ -17,7 +17,7 @@ export interface StatefulAlertProps {
 }
 
 export const statefulPropTypes = {
-    variant: PropTypes.oneOf(['error', 'info', 'message', 'success']).isRequired,
+    variant: PropTypes.oneOf(['error', 'danger', 'info', 'message', 'success']).isRequired,
```
Comment (jwbennet) [inline src/components/Alert/common.ts]: I think we should go ahead and remove the `error` variant as long as we mention it in the commit message for the release notes.

**Comment by jolamar** [comment]
New PR up at #141 for 1.1.0 release

**Pull request closed without merging**